src/lib.rs

```diff
@@ -2,6 +2,8 @@ use imgui::{
     Context, DrawCmd::Elements, DrawData, DrawIdx, DrawList, DrawVert, TextureId, Textures,
 };
 use smallvec::SmallVec;
+use std::error::Error;
+use std::fmt::{Display, Formatter};
```

Comment:
With `fmt` and `io`, you should import the modules directly then do `fmt::Formatter` and such as these modules have a lot of duplicates of prelude stuff.

Reply:
Thanks for the feedback, adjusted my commit